<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="8,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="lbl_Cam">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Brightness:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="Slider_brightness">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="btnWebCam">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>WebCam</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCapture">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Capture</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,7,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,5">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="lbl_Avatar">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>3</number>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="btnAvatar">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>CapAvatar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>25</number>
           </property>
           <property name="rightMargin">
            <number>7</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_ID">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="fi_ID">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lbl_HoTen">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Họ Tên:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="fi_HoTen">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lbl_ChuyenNganh">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Chuyên Ngành:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="fi_ChuyenNganh">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lbl_Lop">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Lớp:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="fi_Lop">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="3">
          <widget class="QLabel" name="lbl_Anh14">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="lbl_Anh10">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_Anh6">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="lbl_Anh9">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="lbl_Anh8">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="lbl_Anh5">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lbl_Anh7">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lbl_Anh2">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="lbl_Anh15">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_Anh11">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="lbl_Anh13">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lbl_Anh12">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lbl_Anh4">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lbl_Anh3">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_Anh1">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbl_Anh16">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="lbl_Anh17">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="lbl_Anh18">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="lbl_Anh19">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="lbl_Anh20">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>C:/Users/Admin/OneDrive/Máy tính/Who1.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="btnSaveDB">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>SaveDB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Exit">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1324</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
